portfolio/src/main/webapp/functions/index.js

```diff
@@ -1,11 +1,26 @@
 const createCommentElement = commentPayload => {
     const username = document.createElement("span");
+    const sentiment = document.createElement("span");
     const comment = document.createElement("p");
     const deleteOption = document.createElement("i");
 
     username.innerHTML = `@ ${commentPayload.username}`;
     username.classList.add("badge", "badge-dark");
     comment.innerHTML = commentPayload.comment;
+
+    const score = parseFloat(commentPayload.sentiment);
+    const emotion =
+        score >= -1 && score <= -0.3 ? "Mad" :
+        score > -0.3 && score <= 0.3 ? "Neutral" :
+        "Happy";
+
+    const badgeToSet =
+        score >= -1 && score <= -0.3 ? "danger" :
```

Comment:
These look to be the same ranges as the "emotion". If that's intentional, consider using the emotion to set the badge instead. That way if you change the ranges in the future, you don't have to do it in both places.

If you think in the future, you might want the emotion and badge ranges to be different, then it's perfectly fine to leave it as-is.